server.js

```diff
@@ -22,7 +23,38 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Start defining your RESTful end points/routes here
+app.get('/catalogue', (req, res) => {
+  res.json(netflixData)
+})
+
+app.get('/type/:type', (req, res) => {
```

Comment:
Don't forget to use RESTful routes. The route should be the pluralised version of the data you're returning. So if this is returning a 'Show', then this endpoint should be `/shows` and then using a query parameter for the type. So ending up with an endpoint like this: `/shows` - returns everything, `/shows?type=movie` - returns only movies